### PR TITLE
Onboarding papers

### PR DESCRIPTION
**Pull request by ntlhui**
Adds onboarding papers

### CODE REVIEW

**Review by TylerFlar — DISMISSED** [review]

**Review by hgrehm — DISMISSED** [review]

File: .github/workflows/onboarding_page_build.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9"
+    - name: Install dependencies
+      run: |
+        sudo sed -i 's/yes/no/g' /etc/initramfs-tools/update-initramfs.conf
```
Comment (ccrutchf) [inline .github/workflows/onboarding_page_build.yml]: It feels weird that you'd need to update the initramfs on machine that isn't going to be rebooted.

File: onboarding_papers/FishSense.bib [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@INPROCEEDINGS{tueller_maddukuri_paxson_et_al_oceans_2021,
```
Comment (ccrutchf) [inline onboarding_papers/FishSense.bib:1]: I think we should add the underwater calibration paper.

File: onboarding_papers/Acoustic_Species_ID.bib
Comment (sprestrelski) [inline onboarding_papers/Acoustic_Species_ID.bib:1]: I would add the [BirdCLEF 2023 Overview](https://scholar.google.com/scholar?q=birdclef+2023+overview) for a wider introduction plus a summary of standard techniques.
```
@article{kahl2023overview,
  title={Overview of BirdCLEF 2023: Automated bird species identification in Eastern Africa},
  author={Kahl, Stefan and Denton, Tom and Klinck, Holger and Reers, Hendrik and Cherutich, Francis and Glotin, Herv{\'e} and Go{\"e}au, Herv{\'e} and Vellinga, Willem-Pier and Planqu{\'e}, Robert and Joly, Alexis},
  journal={Working Notes of CLEF},
  year={2023}
}
```

Comment (Sean1572) [reply onboarding_papers/Acoustic_Species_ID.bib:1]: Perhaps birdclef 2020 for summary of Peru classification troubles

File: .github/workflows/onboarding_page_build.yml (hunk 1 above)
Comment (ntlhui) [reply .github/workflows/onboarding_page_build.yml]: Removing in 883f962ecea8c28fc00f8ffd217d6569a0b242f6

File: onboarding_papers/FishSense.bib (hunk 2 above)
Comment (ntlhui) [reply onboarding_papers/FishSense.bib:1]: Fixed in 1559885251d7175f08bf7ba9d159c33efc95574a

File: onboarding_papers/Acoustic_Species_ID.bib
Comment (ntlhui) [reply onboarding_papers/Acoustic_Species_ID.bib:1]: Fixed in cbcee3c78cf09f486462dfd8df3740aba08dbae7

**Review by sprestrelski — DISMISSED** [review]

**Review by ccrutchf — DISMISSED** [review]

**Review by hgrehm — DISMISSED** [review]

**Review by TylerFlar — DISMISSED** [review]

**Review by Jasonnyang — APPROVED** [review]

**Pull request merged**